### DISCUSSION TITLE
telegram真的太好丸辣！

### DISCUSSION THREAD

**Discussion by hdjsjjdgrhj** [share121/inter-knot#5041] [其他]
![Screenshot_2024-09-17-11-16-19-708_org telegram messenger](https://github.com/user-attachments/assets/c42accbf-be52-40f7-8b67-3d7fb93e1a9a)
telegram真的太好丸辣！不仅有优质的资源，还有老司机必备的神器。

**Comment by capacitor1** [comment]
# TG是真的会限速🤔
![image](https://github.com/user-attachments/assets/415cf20e-be8b-422e-95fa-a5d029a85dbb)
导了3天没导完

**Reply by Xxlllllll** [reply]
破解软件群地址可以分享么(●'◡'●)

**Comment by JiuXiat** [comment]
禁止涩涩

**Reply by Yizakl** [reply]
em，算吗？（我不知道

**Comment by JT9364** [comment]
笑死，我卡在了手机验证码这一关

**Reply by Xxlllllll** [reply]
下载一个telegram X可以解决

**Comment by dzqwe** [comment]
我一也样  
![1145](https://s3.bmp.ovh/imgs/2024/09/17/1a9590fc8d36f55f.jpg)

**Reply by Orangered81** [reply]
bro拿来开盒是吧（

**Reply by Allium0** [reply]
我嘞个梓澪妙妙屋啊😁

**Reply by hsd221** [reply]
我超，盒

**Comment by Mollsity** [comment]
这玩意我纯拿来当盒武器😡

**Reply by dzqwe** [reply]
LOL(笑

**Comment by dzqwe** [comment]
不过咱们最好还是不用盒武器  
毕竟咱们上网不就是为了SS吗  
LOL（笑

**Comment by OUOnrbp** [comment]
之前见过别人截图，用来搜小说，都不知道是啥，原来这个界面叫这个名字，

**Comment by ghost** [comment]
我玩电报也大约快两年了吧从这里提醒大家一句别人不认识的人给你发的东西尽量别点开不然不知道里面有什么东西，万一对方通过黑产的那些不正当技术手段控制了你的设备那就不好玩了
![Screenshot_2024-09-24-17-56-34-177_org telegram group](https://github.com/user-attachments/assets/efa67f6c-19ea-4512-8325-4883cf366dd6)

**Comment by firefallen** [comment]
这玩意一天到晚connecting

**Comment by real-strange** [comment]
我为什么收不到手机验证码<img src="https://bbs-static.miyoushe.com/static/2023/11/17/e0564870de2f3d1c39dfc41a8dcd9d2b_383241188874318913.png" alt="芙宁娜-好困难" width="75" >

**Comment by real-strange** [comment]
都上哪整的这么多《好》群啊？<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by snivm** [comment]
自用机场，稳定，按流量计费，不限时长，流量用完前不会重置，每月不会刷新，支持​Netfli​​和chatgpt,最低​1元套​​，mojie.me/#/register?code=XdLgjGwe​，挂梯子打开

**Comment by zgyhim** [comment]
个人觉得xmpp比较好
和tg界面几乎一模一样~~~tg底层协议可能就是xmpp~~~,分布式服务，匿名性高。注册方便，不用手机号。而且免翻

<img src="https://bbs-static.miyoushe.com/static/2024/08/07/86c603f895bd4e13b331508610a488fc_4608730387935880969.png" alt="流浪者-默认同意" width="75" >

**Comment by hsd221** [comment]
原来大家都是拿tg瑟瑟啊😦

**Comment by mxym-fy** [comment]
tg确实好用啊，而且也不会占太多空间
